### ISSUE TITLE
[Request] Special character support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AmatCoder/mednaffe#11]
```
What steps will reproduce the problem?
1. Use a rom directory containing the ﻼ character
2. Add directory to mednaffe
3. Try to browse roms in the directory

What is the expected output? What do you see instead?
The directory name will appear on the right instead of the left and no roms 
will be found inside the directory, regardless of recursive setting.

What version of the product are you using? On what operating system?
mednaffe-0.6 with mednafen-0.9.32-wip-win32 in Windows 7 64bit

Please provide any additional information below.
I know most people will never use special characters, so this is not a major 
issue, but the "ﻼ" character (I believe it is called ariel) is one of the 
very few characters that comes after "Z" and "z". Even Windows handles this 
character in a strange manner, so if it is too much trouble, don't bother 
fixing this issue. I haven't tried other special characters, but I do know "_" 
(underscore) does work properly. 


```

Original issue reported on code.google.com by `StarrySk...@gmail.com` on 20 Dec 2013 at 8:35

**Comment by TathagataDM** [comment]
It seems that Mednafen now how unicode support if I understand correctly, would it now be possible for Mednaffe to implement it?

**Comment by AmatCoder** [comment]
Yes, mednafen emulator has Unicode support on Windows since version 1.21.x.

And mednaffe has Unicode support on Windows since 0.9.x, but some bug prevents it to be useful (mednafen executable is not detected when a non-ASCII character is used in the path).

**Comment by AmatCoder** [comment]
Fixed with released version [0.9.2](https://github.com/AmatCoder/mednaffe/releases/latest)
